sdk/patina_macro/src/component_macro.rs

```diff
@@ -140,36 +85,38 @@ impl Parse for Component {
 }
 
 /// The testable version of the `component` macro that uses proc_macro2::Tokenstreams.
+// Note: Marked as dead code since it's only used by tests.
+#[allow(dead_code)]
 pub(crate) fn component2(item: proc_macro2::TokenStream) -> proc_macro2::TokenStream {
```

Comment:
Is this allow(dead_code) old?

I assume this has to be used in the actual macro, otherwise we don't have the IntoComponent implementation and the ComponentInput implementation.